site/tryit/README.md

```diff
@@ -35,37 +39,50 @@ First annotate your docker bridge local network address. Usually it's `172.17.0.
 
 
 ::: warning
-NOTE: if you have an ssh server running locally you should stop it
+If you have an **ssh server** running locally you should **stop** it
 :::
 
 ```
 docker run -d --name gitea -v /path/to/gitea-data:/data -p 3000:3000 -p 22:22 gitea/gitea:latest
 ```
-
-This will save your gitea data inside your host `/path/to/gitea` directory
+where:
+   * `-v=path/to/gitea-data` is your gitea data directory
+   * `-p 3000:3000` is the gitea web port provisioned
+   * `-p 22:22` is the gitea ssh port provisioned
+   
 
 #### Setup gitea
 
 
-* Access gitea on `http://172.17.0.1:3000`
-* In the initial setup page you should change:
+**1.** Access gitea on `http://172.17.0.1:3000`
```

Comment:
Doing this (bold numbers) will break the html list behavior making them paragraphs. Just use `1.` `2.`. If you really want them bold (but I'm not sure it's needed) change the theme.